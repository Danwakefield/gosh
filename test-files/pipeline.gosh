echo "2 test cases"
echo "SUXXESS 1" | tr X C
echo "" | true ${A=FAIL 2}
true ${A=SUCCESS 2} # This only assigns if A does not already have a value
echo $A
