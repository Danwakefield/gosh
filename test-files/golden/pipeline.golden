2 test cases
SUCCESS 1
SUCCESS 2
